subprojects/plugin-use/src/main/java/org/gradle/plugin/use/internal/DefaultPluginRequestApplicator.java

```diff
@@ -85,61 +88,101 @@ public DefaultPluginRequestApplicator(
     }
 
     @Override
-    public void applyPlugins(final PluginRequests requests, final ScriptHandlerInternal scriptHandler, @Nullable final PluginManagerInternal target, final ClassLoaderScope classLoaderScope) {
-        if (target == null || requests.isEmpty()) {
+    public void applyPlugins(final PluginRequests requests, PluginRequests autoAppliedPlugins, final ScriptHandlerInternal scriptHandler, @Nullable final PluginManagerInternal target, final ClassLoaderScope classLoaderScope) {
```

Comment:
This would mean we have to cast it everywhere where this method is called or propagate the change. There are a lot of places where this is used, I'm not sure what is the best way to handle it.